t/plugin/chaitin-waf-reject.t

```diff
@@ -0,0 +1,141 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+no_long_string();
+no_root_location();
+
+add_block_preprocessor(sub {
+    my ($block) = @_;
+
+    my $stream_default_server = <<_EOC_;
+    server {
+        listen 8088;
+        listen 8089;
+        content_by_lua_block {
+            require("lib.chaitin_waf_server").reject()
+        }
+    }
+_EOC_
+
+    $block->set_value("extra_stream_config", $stream_default_server);
+    $block->set_value("stream_conf_enable", 1);
+
+    # setup default conf.yaml
+    my $extra_yaml_config = $block->extra_yaml_config // <<_EOC_;
+apisix:
+  stream_proxy:                 # TCP/UDP L4 proxy
+   only: true                  # Enable L4 proxy only without L7 proxy.
+   tcp:
+     - addr: 9100              # Set the TCP proxy listening ports.
+       tls: true
+     - addr: "127.0.0.1:9101"
+   udp:                        # Set the UDP proxy listening ports.
+     - 9200
+     - "127.0.0.1:9201"
+plugins:
+    - chaitin-waf
```

Comment:
The plugin has been enabled by default, there is no need to add it separately

Reply:
Done